charactersheet/charactersheet/templates/dm/encounter_sections/point_of_interest.tmpl.html

```diff
@@ -0,0 +1,189 @@
+<div class="h4">
+  Points of Interest<br />
+  <small>Keep track of all of the interesting locations and
+    objects in the world.</small>
+</div>
+<table class="table table-responsive table-ac-bordered table-hover">
+  <thead>
+    <tr>
+      <th data-bind="click: function(){sortBy('name');}">
+        Name
+        <span data-bind="css: sortArrow('name')"></span>
+      </th>
+      <th data-bind="click: function(){sortBy('description');}">
+        Description
+        <span data-bind="css: sortArrow('description')"></span>
+      </th>
+      <th>
+        <a data-toggle="modal"
+           data-target="#addPointOfInterest" href="#">
+          <i class="fa fa-plus fa-color"></i>
+        </a>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+  <!-- ko foreach: filteredAndSortedPointsOfInterest -->
+    <tr class="clickable">
+      <td data-bind="text: name, click: $parent.editPointOfInterest"
+          href="#"
+          data-toggle="modal"
+          data-target="#viewPointOfInterest"></td>
+      <td data-bind="text: description, click: $parent.editPointOfInterest"
```

Comment:
Should we make a short description label for this? Similar to magic items and items descriptions?